h/services/__init__.py

```diff
@@ -26,6 +27,8 @@ def includeme(config):
     config.register_service_factory('.user_password.user_password_service_factory', name='user_password')
     config.register_service_factory('.user_signup.user_signup_service_factory', name='user_signup')
 
+    config.add_directive('add_annotation_link_generator',
+                         '.links._add_annotation_link_generator')
```

Comment:
Shouldn't we make this function public now that it's referenced here? Maybe we can just make the public function get the config instead of the registry directly.